wire-ios/Wire-iOS/Sources/UserInterface/MainController/SidebarViewControllerBuilder.swift

```diff
@@ -34,10 +34,12 @@ struct SidebarViewControllerBuilder {
         let legalHoldIndicatorViewDesign = LegalHoldIndicatorViewDesign()
 
         let sidebarViewController = SidebarViewController(
-            accountImageView: { accountImage, availability in
+            accountImageView: { accountImage, availability, showNotificationsBadge in
                 AccountImageViewRepresentable(
                     source: accountImage.mapToAccountImageSource(),
-                    availability: availability?.mapToAccountImageAvailability()
+                    availability: availability?.mapToAccountImageAvailability(),
+                    // TODO: [WPB-11968] Use the actual value based on team membership, available API version & UserDefaults,
```

Comment:
Ticket WPB-11968 is marked as in code review. Is there another PR which solves this TODO?

Reply:
The comment is outdated.